### PR TITLE
Prune old dependencies, update pkgs, drop graphiql

### PR DESCRIPTION
**Pull request by krzysztofwolski**
- fixed missing graphs
- up to date deps versions
- drop graphiql

### CODE REVIEW

**Comment by krzysztofwolski** [comment]
Build maser: 6m 25s
Build time pr: 3m 25s

Not sure if thats removing deps or new docusaurus version, but I'm not complaining.

**Review by korycins — APPROVED** [review]

File: website/versioned_docs/version-2.11/developer/products.mdx [hunk 1]
```diff
@@ -54,18 +75,44 @@ The `products` query gives the ability to filter the results. Use the optional `
 
 Here is an example query that looks for the first 3 products containing the term "juice" in the title or description:
 
-<GraphQL
-  height={20}
-  query={`{
+```json
```
Comment (krzysztofwolski) [inline website/versioned_docs/version-2.11/developer/products.mdx:78]:
```suggestion
```graphql
```

File: docs/developer/metadata.mdx [hunk 2]
```diff
@@ -55,9 +53,8 @@ type MetadataItem {
 
 Here's an example query that fetches the metadata of the first product:
 
-<GraphQL
-  height={25}
-  query={`{
+```graphql
```
Comment (patrys) [inline docs/developer/metadata.mdx:56]: Would this be more readable as tabs?

**Review by patrys — APPROVED** [review]

File: docs/developer/metadata.mdx [hunk 3]
```diff
@@ -70,8 +67,36 @@ Here's an example query that fetches the metadata of the first product:
...
+            "metadata": [
+              {
+                "key": "vatlayer.code",
+                "value": "standard"
+              },
+              {
+                "key": "vatlayer.description",
+                "value": "standard"
+              }
+            ]
```
Comment (patrys) [inline docs/developer/metadata.mdx:92]: I think we can find better examples of metadata?

File: docs/developer/metadata.mdx (hunk 2 above)
Comment (krzysztofwolski) [reply docs/developer/metadata.mdx:56]: Tried this and IMO readability goes down - I prefer to see request and response the same time, it's easier for me to understand when I don't need to switch the tabs

File: docs/developer/metadata.mdx (hunk 3 above)
Comment (krzysztofwolski) [reply docs/developer/metadata.mdx:92]: IMO it's a good example of using metadata for app synchronization. @patrys  Removing description would make this better? (removed duplication)

Comment (patrys) [reply docs/developer/metadata.mdx:92]: I was thinking about using something self-explanatory like `pim.externalReference` and `tax.group` maybe?

**Review by patrys — APPROVED** [review]

**Pull request merged**